webpack-helper.js

```diff
@@ -44,12 +47,14 @@ module.exports = function(config, dev) {
 
   config.module = config.module || {};
   config.module.loaders = config.module.loaders || [];
+  config.module.preLoaders = config.module.preLoaders || [];
 
   if (!dev) {
-    ensureLoader(config.module.loaders, /\.js$/, 'webpack-strip-block');
+    ensureLoader(config.module.preLoaders, /\.js$/, 'webpack-strip-block');
   }
   ensureLoader(config.module.loaders, /\.mustache$/, 'tungsten_template');
   ensureLoader(config.module.loaders, /\.json$/, 'json-loader');
+  ensureLoader(config.module.loaders, /\.jsx?$/, 'babel');
```

Comment:
Did you mean to add the exclusion regex from https://github.com/wayfair/tungstenjs/pull/54/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR34 here?